### ISSUE TITLE
[RESEARCH] How do we auto-close toasts in an accessible manner?

### ISSUE THREAD

**Issue by Blackbaud-AlexKingman** [blackbaud/skyux-toast#23]
@Blackbaud-SteveBrush commented on [Fri Jun 29 2018](https://github.com/blackbaud/skyux2/issues/1779)
